API/DeleteWordLinkBackTranslation.php

```diff
@@ -0,0 +1,42 @@
+<?php
+require_once('utils/Model.php');
+session_start();
+
+
+if ($_SERVER['REQUEST_METHOD'] !== 'POST') {
+    RespondWithError(405, "Request must be a POST");
+}
+
+if (!(
+    array_key_exists('PhoneId', $_POST) &&
+    array_key_exists('term', $_POST))
+) {
+    RespondWithError(400, 'This endpoint requires PhoneId & term.');
+}
+
+$androidId = trim($_POST['PhoneId']);
+$term = trim($_POST['term']);
+
+$model = new Model();
+# check if project exists for the given androidId
+$projectId = $model->GetProjectId($androidId);
+
+if (!$projectId) {
+    RespondWithError(404, "");
```

Comment:
Can you use the same error message for missing project id's? 
Please register a project using /API/RegisterPhone.php before using /API/UploadWordLinkBackTranslation.php

Reply:
I can do that but it doesn't feel good to ask a user to create upload wordlink before deleting it.

If we put end user shoes, then I am trying to delete a wordlink but I have entered incorrect project. So I got 404.

Lemme know if you don't then I would ahead and replicate the same message.

Reply:
Great point. So maybe something like "Please double check the PhoneId. A project with $androidId was not found."

Reply:
@johntseng Error message updated.